Geo-Cordinate Locator/Requirements.txt

```diff
@@ -0,0 +1,3 @@
+tkinter==8.6.10
+geopy==2.0.0
+So, you can install them all the packages used at once pip install -r requirements.txt
```

Comment:
Remove 
`So, you can install them all the packages used at once pip install -r requirements.txt` this line I wrote that for you

Reply:
oh achchaa. Got it

Reply:
done